tests/containers/zypper_docker.pm

```diff
@@ -34,7 +34,7 @@ sub run {
     # install zypper-docker and verify installation
     zypper_call('in zypper-docker');
     validate_script_output("zypper-docker -h", sub { m/zypper-docker - Patching Docker images safely/ }, 180);
-    my $testing_image = 'opensuse/leap';
+    my $testing_image = 'registry.opensuse.org/opensuse/leap';
```

Comment:
```suggestion
    my $testing_image = get_var('REGISTRY','registry.opensuse.org') . '/opensuse/leap';
```

Reply:
For openSUSE images we use registry.opensuse.org. But for alpine and other non-suse images we use the defined by var `REGISTRY`.